user_tools/src/spark_rapids_pytools/cloud_api/dataproc_gke.py

```diff
@@ -196,5 +196,4 @@ class DataprocGkeSavingsEstimator(DataprocSavingsEstimator):
     def _get_cost_per_cluster(self, cluster: ClusterGetAccessor):
         master_cost = self._calculate_group_cost(cluster, SparkNodeType.MASTER)
         workers_cost = self._calculate_group_cost(cluster, SparkNodeType.WORKER)
-        dataproc_gke_cost = self.price_provider.get_container_cost()
-        return master_cost + workers_cost + dataproc_gke_cost
+        return master_cost + workers_cost
```

Comment:
For Dataproc GKE this might not be correct. `Dataproc GKE` inherits methods from `Dataproc` and overrides some of the methods. For the above changes, definition of `get_container_cost()` in Dataproc GKE should be removed.

1. Method `get_container_cost()` called within `_calculate_group_cost()` should be from Dataproc, not Dataproc GKE.  
2. Dataproc GKE has a seperate cluster management fee irrespective of node size. We still need to add the value from `__get_gke_container_cost()`.

Reference -

https://github.com/NVIDIA/spark-rapids-tools/blob/ebca53035da1ac790a850a58fe21176c741060eb/user_tools/src/spark_rapids_pytools/pricing/dataproc_gke_pricing.py#L29-L36

Reply:
To confirm, I can fully remove this line: https://github.com/NVIDIA/spark-rapids-tools/blob/ebca53035da1ac790a850a58fe21176c741060eb/user_tools/src/spark_rapids_pytools/pricing/dataproc_gke_pricing.py#L30. 

And then remove my changes to the GKE pricing code to let it only use the `__get_gke_container_cost()` calculation for its pricing.

Reply:
We need to remove the `get_container_cost()` function from `dataproc_gke_pricing.py` entirely. Otherwise, when `_calculate_group_cost()` is called, it will use the method from `dataproc_gke_pricing.py` instead of `dataproc_pricing.py`.

For including cluster management cost, we can create a new function `get_cluster_management_cost()` in `price_provider.py`. This function should have an implementation in `dataproc_gke_pricing.py`. Now, we can call `get_cluster_management_cost()` from `_get_cost_per_cluster()` to use this cost.

Reply:
OK, I simplified changes to only be in Dataproc path inside the `get_cost_per_cluster` function.  Thanks for your help.

Reply:
Thanks @mattahrens